### PR TITLE
feat: Improve UX of space filtering and space bookmarking - MEED-1578 

### PR DESCRIPTION
**Pull request by MayTekayaa**
This change will improve the UX of space filtering and space bookmarking.

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/common/components/FavoriteButton.vue [hunk 1]
```diff
@@ -35,7 +35,7 @@
             size="16">
             {{ favoriteIcon }}
           </v-icon>
-          <span class="text-color mt-1">{{ favoriteLabel }}</span>
+          <span class="text-color mt-1" :class="extraClass">{{ favoriteLabel }}</span>
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/common/components/FavoriteButton.vue]:
```suggestion
          <span :class="extraClass" class="text-color mt-1">{{ favoriteLabel }}</span>
```

File: webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue [hunk 2]
```diff
@@ -89,15 +89,13 @@
...
+                  <v-list-item-title>
                     <exo-space-favorite-action
                       v-if="favoritesSpaceEnabled"
                       :is-favorite="space.isFavorite"
                       :space-id="space.id"
-                      class="ms-1 me-2 mb-1" />
-                    <span class="mx-2">
-                      {{ favoriteTitleToDisplay }}
-                    </span>
+                      :extra-class="'ms-4'"
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue]:
```suggestion
                      extra-class="ms-4"
```

Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/spaces-list/components/ExoSpaceCardFront.vue]: done!

File: webapp/portlet/src/main/webapp/vue-apps/common/components/FavoriteButton.vue (hunk 1 above)
Comment (MayTekayaa) [reply webapp/portlet/src/main/webapp/vue-apps/common/components/FavoriteButton.vue]: done!

**Review by boubaker — APPROVED** [review]

**Pull request merged**